### ISSUE TITLE
Creating VPC Network as admin there is no option to select Domain and Account

### ISSUE THREAD

**Issue by msinhore** [apache/cloudstack#10121]
##### ISSUE TYPE

 * Improvement Request

##### COMPONENT NAME

~~~
VPC
~~~

##### CLOUDSTACK VERSION

~~~
4.20.0
~~~

##### SUMMARY

When creating a VPC Network as an Administrator, there is no option to select a Domain or Account. Adding this functionality would improve consistency with the “Add Network” feature for Guest Networks, where it is possible to associate the network with a specific Domain or Account. 

##### STEPS TO REPRODUCE

~~~
Log in as an Administrator.
Navigate to Network > VPC > Add VPC.
Attempt to create a VPC Network.
~~~

##### EXPECTED RESULTS

~~~
Administrators should be able to assign a Domain and Account while creating a VPC Network, similar to the process for creating  Isolated Networks.
~~~

##### ACTUAL RESULTS

~~~
There is no option to select Domain and Account. 
~~~

**Comment by kiranchavala** [comment]
@msinhore the option is present in the API, there is account and domainid parameters

https://cloudstack.apache.org/api/apidocs-4.20/apis/createVPC.html

Its just an UI improvement

**Comment by weizhouapache** [comment]
yes @kiranchavala 

#8919 has improved some forms

**Comment by DaanHoogland** [comment]
can we close as fixed in #8919, @msinhore @kiranchavala ?

**Comment by weizhouapache** [comment]
@DaanHoogland 
it seems #8919 only improved the form for Isolated and L2, not VPC

**Comment by BryanMLima** [comment]
@weizhouapache, that's correct, I created the PR #10124 to address this.

**Comment by DaanHoogland** [comment]
fixed by #10124
